src/common/pf_alarm.c

```diff
@@ -978,12 +978,12 @@ static int pf_alarm_apmx_activate (pnet_t * net, pf_ar_t * p_ar)
             p_ar->apmx[ix].frame_id = PF_FRAME_ID_ALARM_HIGH;
          }
 
-         if (p_ar->apmx[ix].p_alarm_q == NULL)
+         /* Input alarm frame queue */
+         if (p_ar->apmx[ix].alarm_receive_q.p_receive_mutex == NULL)
          {
-            p_ar->apmx[ix].p_alarm_q = os_mbox_create (PNET_MAX_ALARMS);
+            p_ar->apmx[ix].alarm_receive_q.p_receive_mutex = os_mutex_create();
```

Comment:
Maybe "mutex" works as name? p_ should be avoid according to coding standard. That it is for the receive queue is given by the containing struct/variable name

Reply:
Done